pipelinewise/fastsync/commons/tap_postgres.py

```diff
@@ -317,9 +323,7 @@ def fetch_current_log_pos(self):
                     'Logical replication not supported before PostgreSQL 9.4'
                 )
 
-        current_lsn = result[0].get('current_lsn')
-        file, index = current_lsn.split('/')
-        lsn = (int(file, 16) << 32) + int(index, 16)
+        lsn = parse_lsn(result[0].get('current_lsn'))
```

Comment:
Is this a `dict.get`? If so, should `parse_lsn` accept `Optional[str]`?

Reply:
Yup, was a copy-pasta from elsewhere. Shouldn't be a `get`.